pkg/grafana/client_test.go

```diff
@@ -320,16 +320,19 @@ func TestGetOrCreateUser(t *testing.T) {
 
 	client := NewMockClient(user, userOrgsRoleMap{})
 
-	orgUser, err := client.GetOrCreateUser("foo")
+	// Existing users, only Login is used for lookup
+	orgUser, err := client.GetOrCreateUser("foo", "", "")
 	assert.NoError(t, err)
 	assert.Equal(t, user, orgUser)
 
 	// New user
-	newUser, err := client.GetOrCreateUser("new")
+	newUser, err := client.GetOrCreateUser("new", "foo", "bar@foo.com")
```

Comment:
Not sure if it is a viable suggestion, but maybe we can create the `new` user before the existing user validation and then lookup the same user right after instead of having to create `foo` before starting the tests?